js/_website/src/lib/templates/gradio/04_helpers/01_eventdata.svx

```diff
@@ -0,0 +1,72 @@
+
+<script lang="ts">
+    import {get_object} from "../../process_json.ts";
+    import ParamTable from "$lib/components/ParamTable.svelte";
+    import ShortcutTable from "$lib/components/ShortcutTable.svelte";
+    import DemosSection from "$lib/components/DemosSection.svelte";
+    import FunctionsSection from "$lib/components/FunctionsSection.svelte";
+    import GuidesSection from "$lib/components/GuidesSection.svelte";
+    import CopyButton from "$lib/components/CopyButton.svelte";
+    import { style_formatted_text } from "$lib/text";
+
+    let obj = get_object("eventdata");
```

Comment:
I think you forgot to add the kwargs thing here 


![Screen Shot 2024-06-25 at 7 59 43 PM](https://github.com/gradio-app/gradio/assets/9021060/9d3b12ac-a67b-4414-a3c5-d7d7b2ecaa24)

Reply:
```suggestion
    let obj = get_object("eventdata");
    obj.parameters[0]["kwargs"] = null;
```